app/controllers/job_runs_controller.rb

```diff
@@ -26,7 +25,7 @@ def create
 
   def download_log
     @job_run = JobRun.find(params[:id])
-    if @job_run.progress_log_file && File.exist?(@job_run.progress_log_file)
+    if @job_run.progress_log_file_exists?
       send_file @job_run.progress_log_file
```

Comment:
added a new helper method on the `JobRun` model to do this check so it can be used elsewhere too